content/items/vtr.md

```diff
@@ -1,17 +1,46 @@
 ---
 title: Verilog to Routing (VTR)
-description: Open-source CAD tools for FPGA architecture and CAD research
+description: Open Source CAD Flow for FPGA Research
 authors: []
 links:
   web: https://verilogtorouting.org
   gh: verilog-to-routing/vtr-verilog-to-routing
+categories: [
+  "Frameworks",
+  "Tools",
+  "Tools:PnRs",
+  "Tools:Synthesizers"
+]
 tags: [
   "synthesis",
+  "packing",
+  "placement",
+  "routing",
+  "sta",
   "verilog",
+  "odin",
+  "ABC",
+  "VPR",
+  "FASM",
+]
+active:
+  from: ????
+licenses: [
+  "MIT"
 ]
 talk: 123
 ---
 
-This is a long description...
-<!--more-->
-... about VTR.
+*"The Verilog-to-Routing (VTR) project is a world-wide collaborative effort to provide a open-source framework for conducting FPGA architecture and CAD research and development. The VTR design flow takes as input a Verilog description of a digital circuit, and a description of the target FPGA architecture."*
+
+It performs:
+* Elaboration & Synthesis (ODIN II)
+* Logic Optimization & Technology Mapping (ABC)
+* Packing, Placement, Routing & Timing Analysis (VPR)
+
+*"to generate FPGA speed and area results. \[...\] VTR can also produce FASM to program some commercial FPGAs (via Symbiflow)."*
+
+References:
+
+- [Versatile Place and Route]({{< ref "/items/vpr" >}} "Versatile Place and Route")
+- [FASM]({{< ref "/items/fasm" >}} "FASM")
```

Comment:
Add references to SymbiFlow and Yosys too :wink:

Reply:
Ok about SymbiFlow. Not sure about Yosys :thinking: VtR uses ODIN II for synthesis and Yosys is already referenced in SymbiFlow.

Reply:
I added SymbiFlow and I updated the dates according to https://github.com/verilog-to-routing/vtr-verilog-to-routing/issues/1691.

Reply:
> Ok about SymbiFlow. Not sure about Yosys 🤔 VtR uses ODIN II for synthesis and Yosys is already referenced in SymbiFlow.

ODIN-II is the tool in VTR as a suite, but it is also possible to use the output from Yosys, usin (E)BLIF. See https://github.com/hdl/awesome/issues/98. Someone might want to use Yosys and VTR, regardless of SymbiFlow.

Reply:
Added (Yosys, check the order)